change/@microsoft-teams-js-19c8476a-7a54-4481-b8cb-811fcd3bab65.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Created unified way for capability functions to communicate with hosts and deprecated old functions used for this purpose. This change is internal to the library and should not affect any callers.",
```

Comment:
Can we add all the functions that are now deprecated with reference to their replacements in this comment for anyone that looks through the changelog of the release this change will go out with?

Reply:
This change shouldn't impact external callers, but until we decide if we're keeping the authenticationCEA part of the change I will leave this as a reminder to update the message about those changes if we keep them.